ext/standard/type.c

```diff
@@ -78,8 +67,10 @@ PHP_FUNCTION(gettype)
 
 				if (type_name) {
 					RETVAL_STRING("resource");
-					break;
+				} else {
+					RETVAL_STRING("resource (closed)");
```

Comment:
The usage of parenthesis introduces more inconsistency and makes it harder to use the output in error messages that already use punctuation. Why not `closed resource` or `unknown resource` (`== unknown type`).